packages/shared/lib/core/wallet/tests/getOutputOptions.test.ts

```diff
@@ -181,6 +181,28 @@ describe('File: getOutputOptions.ts', () => {
         expect(output).toStrictEqual(expectedOutput)
     })
 
+    it('should return output options for nft to layer 2', () => {
+        newTransactionDetails = {
+            type: NewTransactionType.NftTransfer,
+            recipient: baseTransaction.recipient,
+            nftId,
+            layer2Parameters,
+        }
+        const output = getOutputOptions(newTransactionDetails)
+
+        const expectedOutput = {
+            recipientAddress: layer2Parameters.networkAddress,
+            amount: addGasBudget('0'),
```

Comment:
Should we be using the functions in the tests? Or define what the output should look like completly?

Reply:
I think we can use functions that have separate unit tests already. Since this is not the case, I removed the functions that don't have it.